crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -1594,6 +1597,9 @@ pub enum SuiObjectArg {
         initial_shared_version: SequenceNumber,
         mutable: bool,
     },
+    // A reference to a Move object that's going to be received in the transaction.
+    #[serde(rename_all = "camelCase")]
+    Receiving(SuiObjectRef),
```

Comment:
Let's not expose this over the rPC until we are ready to use it on one of the networks.